virtualization-and-containers/material/vagrant_slides.md

```diff
@@ -0,0 +1,197 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+  font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# Vagrant
+
+---
+
+## Introduction
+
+- Remember:
+  > If you can't git diff a file format, it's broken.
+- Infrastructure as code
+  > Vagrant is a tool for building and managing virtual machine environments in a single workflow.
+  [https://www.vagrantup.com/intro](https://www.vagrantup.com/intro)
+- Initial release 2010 by Mitchell Hashimoto, now [HashiCorp](https://www.hashicorp.com/)
+- HashiCorp develops open-source DevOps/Cloud tools like [Vagrant](https://www.vagrantup.com/), [TerraForm](https://www.terraform.io/)...
+
+---
+
+## What for?
+
+- Developing software
+  - Set up consistent development environment
+  - Share environment with others
+- Testing software, workflows...
+  - Disposable environments
+  - Consistent workflows
+- More general
+  - Easy way to setup virtual machine
+
```

Comment:
Not sure what your plan, but here could be another occasion to mention "Reproducibility"

Reply:
I think it is always on my mind, but it is better to add it there as well.